### ISSUE TITLE
URDF describes IMU frames for non-imu camera, resulting in "'imu_offset_x' is not defined" error 

### ISSUE THREAD

**Issue by AlessioMorale** [luxonis/depthai-ros#96]
Running depthai_descr.urdf.xacro setting `camera_model:=OAK-D-LITE` fails with the following error:

```
name 'imu_offset_x' is not defined 
when evaluating expression 'imu_offset_x'
when instantiating macro: depthai_camera (/home/alessio/code/ros/depthai/src/luxonis/depthai-ros/depthai_bridge/urdf/include/depthai_macro.urdf.xacro)
```
Steps to reproduce:

- setup an empty ros workspace as described in [README](https://github.com/luxonis/depthai-ros/blob/main/README.md) 
- catkin build
- catkin source
- roslaunch depthai_bridge urdf.launch camera_model:=OAK-D-LITE # immediately fails with the error above

**Comment by saching13** [comment]
Oh my bad. I will add the IMU position for the rest.

**Comment by AlessioMorale** [comment]
I fixed this with #97 by enabling IMU related frames using the `has_imu` property. Currently only for OAK-D (that's the only one defining imu_* variables).

**Comment by saching13** [comment]
Actually post kickstarted verision of OAK-D-Lite has IMU. Ididn't get time to add it here yet. But I agree. I will add your PR for now and upgrade it later. 
THanks for the PR.
